python/tests/test_common_estimator.py

```diff
@@ -425,6 +455,31 @@ def assert_model(model: SparkRapidsMLDummyModel) -> None:
         model_loaded = SparkRapidsMLDummyModel.load(model_path)
         assert_model(model_loaded)
 
+        dummy2 = dummy.copy()
+        assert dummy2.cuml_params["a"] == 100
+        with pytest.raises(
+            Exception,
+            match="assert {'a': 9876.0, 'k': 4, 'x': 40.0} == {'a': 100, 'k': 4, 'x': 40.0}",
+        ):
+            dummy2.fit(df, {dummy2.alpha: 9876.0})
+        assert dummy2.cuml_params["a"] == 100
+        assert dummy2.getOrDefault(dummy2.alpha) == 100
+
+        dummy3 = SparkRapidsMLDummy(
+            inputCols=input_cols,
+            a=100,
+            num_workers=gpu_number,
+            partition_num=ceiling_division(m, max_records_per_batch),
+            m=m,
+            n=n,
+            runtime_check=False,  # don't assert on the runtime.
+        )
+        model3 = dummy3.fit(df, {dummy3.alpha: 9876.0})
+        assert dummy3.cuml_params["a"] == 100
+        assert dummy3.getOrDefault(dummy3.alpha) == 100
+        assert model3.cuml_params["a"] == 9876.0
+        assert model3.getOrDefault(model3.alpha) == 9876.0
```

Comment:
If `dummy3` is just for testing that `model.alpha == 9876.0`, can we just add the model and asserts to `dummy2` tests above (to avoid another section of tests).

Reply:
Otherwise, LGTM

Reply:
the `dummy2.fit` will raise an exception when enabling the runtime check, since we're passing alphas=9876 into fit function. In that case, we can't get the model.